buku.py

```diff
@@ -2088,6 +2088,8 @@ def load_chrome_database(self, path, unique_tag, add_parent_folder_as_tag):
 
         roots = data['roots']
         for entry in roots:
+            if isinstance(roots[entry], str):
```

Comment:
What is this change? Please add a comment.

Reply:
see below please